src/Gaffer/TweakPlug.cpp

```diff
@@ -112,17 +112,39 @@ vector<T> tweakedList( const std::vector<T> &source, const std::vector<T> &tweak
 {
 	vector<T> result = source;
 
+	struct HashFunc
+	{
+		size_t operator()(const T& x) const
+		{
+			IECore::MurmurHash h;
+			h.append( x );
+			return h.h1();
+		}
+	};
+
+	std::unordered_set<T, HashFunc> tweakSet;
+	if constexpr( std::is_same_v< T, bool > )
+	{
+		for( auto i : tweak )
+		{
+			tweakSet.insert( i );
+		}
+	}
+	else
+	{
+		for( const auto &i : tweak )
+		{
+			tweakSet.insert( i );
+		}
+	}
```

Comment:
I think this can be simplified to `std::unordered_set<T, HashFunc> tweakSet( tweak.begin(), tweak.end() );`.

Reply:
Fixed. Should be able to merged once the tests pass.